docker/docker-compose.yml

```diff
@@ -286,7 +287,8 @@ services:
       retries: 60  # Increase retries to allow more time for Elasticsearch to be ready
 
 volumes:
-  fuel-streams-publisher-db:
+  fuel-streams-publisher-db-mainnet:
```

Comment:
Question: why do we want any mainnet stuff in the docker-compose. Mainnet will always run on k8s and we also dont want anyone using docker-compose to push messages to mainnet as a setup, right ?

Reply:
already discussed. closing.